test/metabase/sync_test.clj

```diff
@@ -26,9 +25,17 @@
 ;; These tests make up a fake driver and then confirm that sync uses the various methods defined by the driver to
 ;; correctly sync appropriate metadata rows (Table/Field/etc.) in the Application DB
 
-(def ^:private sync-test-tables
+(driver/register! ::sync-test, :abstract? true)
+
+(def ^:dynamic *supports-schemas?*
+  "Whether the database supports schemas."
+  true)
```

Comment:
I added this dynamic var because previously there was no branching on whether a database supports schemas or not. Now there is [here](https://github.com/metabase/metabase/pull/41588/files#diff-fcbc67149f48554b35e85208009bc7cd27be9a752357f774eee1e652cbe0f9baR80-R81) and [here](https://github.com/metabase/metabase/pull/41588/files#diff-fcbc67149f48554b35e85208009bc7cd27be9a752357f774eee1e652cbe0f9baR92-R93).